### PR TITLE
Do not display error log for initial admin creation

### LINKED ISSUE #15789: "Failed to add user 'admin' ..." should not be an ERROR
### Area

dist/quarkus

### Describe the bug

When using the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` keycloak logs this on startup:

```
ERROR [org.keycloak.services] (main) KC-SERVICES0010: Failed to add user 'admin' to realm 'master': user with username exists
```

In other words: This error always appears except for the very first startup of keycloak. When deploying to kubernetes, there is no way to set these environment variables only for the first deployment.

We forward all `ERROR` logs to our 24/7, so this is annoying. I fail to see why this expected situation should be logged as an "error".

### Version

20.0.1

### Expected behavior

Should be logged as an INFO.

### Actual behavior

Log-Level is ERROR.

### How to Reproduce?

Start a keycloak-container that uses the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` while using a persistent data storage. Then stop and restart the container.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mabartos**
Closes #15789

@vmuzikar Could you please check it?

### CODE REVIEW

File: services/src/main/java/org/keycloak/services/ServicesLogger.java [hunk 1]
```diff
@@ -467,4 +467,8 @@ public interface ServicesLogger extends BasicLogger {
     @Message(id=106, value="Created script engine '%s', version '%s' for the mime type '%s'")
     @Once
     void scriptEngineCreated(String engineName, String engineVersion, String mimeType);
+
+    @LogMessage(level = DEBUG)
```
Comment (vmuzikar) [inline services/src/main/java/org/keycloak/services/ServicesLogger.java:471]: I'd make it `INFO` just as a gentle reminder to a user that they are trying to create an admin user again.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@mabartos Thanks! In general LGTM, added two minor-ish comments.

File: services/src/main/java/org/keycloak/services/ServicesLogger.java (hunk 1 above)
Comment (vmuzikar) [reply services/src/main/java/org/keycloak/services/ServicesLogger.java:471]: @pedroigor Does `INFO` work for you?

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java [hunk 2]
```diff
@@ -63,4 +67,30 @@ void testBuildAndStart(KeycloakDistribution dist) {
...
+        assertTrue(result.getOutput().contains("Added user '" + initialUsername + "' to realm 'master'"),
+                () -> "The Output:\n" + result.getOutput() + "doesn't contains the expected string.");
+
+        dist.setEnvVar("KEYCLOAK_ADMIN", nextUsername);
+        dist.setEnvVar("KEYCLOAK_ADMIN_PASSWORD", password);
+        CLIResult cliResult = dist.run("start-dev", "--log-level=debug");
+
+        cliResult.assertMessage("Skipping create admin user. Admin already exists in realm 'master'.");
+        cliResult.assertStartedDevMode();
+    }
```
[the remark this answers is not in the document]
Comment (mabartos) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java:95]: Yep, the separation of the test cases seems reasonable.

File: services/src/main/java/org/keycloak/services/ServicesLogger.java (hunk 1 above)
Comment (mabartos) [reply services/src/main/java/org/keycloak/services/ServicesLogger.java:471]: I'm not sure. Do we really need to remind a user about the fact we did nothing? :) IMHO, it's sufficient to inform the user only about the successful creation of the admin and then about failed ops. I'd keep the `DEBUG` here. @vmuzikar WDYT?

**Review by pedroigor — APPROVED** [review]

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (pedroigor) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java:95]: For this one, I would not worry about testing using a container. Sounds redundant to me as we would be only checking if env vars are recognized, which does not make much sense.

File: services/src/main/java/org/keycloak/services/ServicesLogger.java (hunk 1 above)
Comment (vmuzikar) [reply services/src/main/java/org/keycloak/services/ServicesLogger.java:471]: @mabartos Ok, fair enough. :) Let's keep `DEBUG`. We display `INFO` if admin is successfully created.

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**